mne/decoding/time_gen.py

```diff
@@ -382,6 +390,8 @@ def _predict_time_loop(X, estimators, cv, slices, predict_mode):
     slices : list
         List of slices selecting data from X from which is prediction is
         generated.
+    predict_function : {'predict', 'predict_proba', 'decision_function'}
+        Specifies prediction function. See scikit-learn.
```

Comment:
I think it shoulnd't be restricted to these, but be restricted to whatever method the estimator has, and that's what the doc should say too. That is what the test is doing too right?